change-camera-controller/src/main/java/com/esri/arcgismaps/sample/changecameracontroller/MainActivity.kt

```diff
@@ -0,0 +1,286 @@
+    )
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        // authentication with an API key or named user is
+        // required to access basemaps and other location services
+        ArcGISEnvironment.apiKey = ApiKey.create(BuildConfig.API_KEY)
+        lifecycle.addObserver(sceneView)
+
+        // create and add a scene with an imagery basemap style
+        val scene = ArcGISScene(BasemapStyle.ArcGISImagery).apply {
+            // add an elevation data source to the base surface
+            baseSurface.elevationSources.add(
+                ArcGISTiledElevationSource(getString(R.string.elevation_service_url))
+            )
+        }
+
+        // graphics overlay for the scene to draw the 3d graphics on
+        val graphicsOverlay = GraphicsOverlay().apply {
+            // set the altitude values in the scene to be absolute
+            sceneProperties.surfacePlacement = SurfacePlacement.Absolute
+            // add the airplane 3d graphic to the graphics overlay
+            graphics.add(airplane3DGraphic)
+        }
+
+        sceneView.apply {
+            // set the scene to the SceneView
+            this.scene = scene
+            // add the graphics overlay to the SceneView
+            graphicsOverlays.add(graphicsOverlay)
+        }
+
+        lifecycleScope.launch {
+            // if the map load failed show an error and return
+            scene.load().onFailure {
+                showError("Failed to load the scene: ${it.message}")
+                return@launch
+            }
+            // set the sceneView viewpoint to the default camera
+            sceneView.setViewpointCamera(defaultCamera)
+            // copy assets to the cache directory if needed
+            copyAssetsToCache(assetFiles, cacheDir, false)
+            // load the airplane model file and update the the airplane3DGraphic
+            loadModel(getString(R.string.bristol_model_file), airplane3DGraphic)
+        }
+
+        // set the click listener for the options dropdown view
+        cameraControllerOptionsView.setOnItemClickListener { parent, _, position, _ ->
+            // get the selected camera mode item
+            val selectedItem = parent.getItemAtPosition(position) as String
+            // get the CameraControllerMode from the selected item
+            CameraControllerMode.getValueOrNull(selectedItem)?.let { mode ->
+                // update the camera controller
+                setCameraControllerMode(mode)
+            }
+        }
+    }
+
+    /**
+     * Loads a [ModelSceneSymbol] from the [filename] in [getCacheDir] and updates the [graphic].
+     */
+    private suspend fun loadModel(filename: String, graphic: Graphic) {
+        val modelFilePath = File(cacheDir, filename)
+        if (modelFilePath.exists()) {
+            // create a new ModelSceneSymbol with the file
+            val modelSceneSymbol = ModelSceneSymbol(modelFilePath.absolutePath).apply {
```

Comment:
```suggestion
            val modelSceneSymbol = ModelSceneSymbol(modelFile.path).apply {
```
Just using `.path` should work fine here